app/components/selected_chip/index.tsx

```diff
@@ -73,16 +69,13 @@ export default function SelectedChip({
             style={style.container}
             testID={testID}
         >
-            {extra && (
-                <View style={style.extraContent}>
-                    {extra}
-                </View>
-            )}
+            {extra}
             <Text
-                style={style.text}
+                style={[style.text, {maxWidth: dimensions.width * 0.70}]}
+                numberOfLines={1}
                 testID={`${testID}.display_name`}
             >
-                {text}
+                {text.replace(' ', '\xa0')}
```

Comment:
why the replace?

Reply:
Funny story. Imagine the text `SomeLongWordHere Short`. Now imagine that the space we have cannot contain `SomeLongWordHere`. So it has to calculate how much space it can show. Then for some reason... that calculation, instead of being the actual available space, it is the length of `Short`. So... instead of having something like `SomeLongWor...` you end up with `SomeL...`.

Doing this replace, we are changing normal spaces by non-breaking spaces. That way, it cannot separate the words to do this calculation, and does the calculation right.

No idea why it does this, and it got me way too long to figure out this workaround, but now it seems to work as it should.

Reply:
This is an incredible find, we should share this more broadly but have you looked into the break strategy props for ios and android? https://reactnative.dev/docs/text#textbreakstrategy-android

0/5 if we want to change it, but we should at least give it a try

Reply:
Setting the textbreakstrategy to 'simple', works better for `SomeLongWordHere Short`, but not so well for `Short SomeLongWordHere` :sweat_smile:

`balanced` seems to solve the issue in most scenarios, but not all of them. I still cannot figure out exactly why. The most consistent solution seems to be the non-breaking spaces.

And even if this was consistent, that would solve for API +24, and I think we support a few behind that, right?

I just added this approach for the `user_item`, where this was also giving some minor trouble, and also to other composed strings. For example (simplified):
```
<Text style={styles.normal}>
   {displayName}
    <Text style={style.dimmed}>
        {` ~${channelName}`}
    </Text>
</Text>
```

Reply:
Thanks for trying